legal-api/src/legal_api/__init__.py

```diff
@@ -67,6 +68,8 @@ def create_app(run_mode=os.getenv('FLASK_ENV', 'production')):
 
     register_shellcontext(app)
 
+    socketio.init_app(app, cors_allowed_origins='*')
```

Comment:
Can we lock the allowed origins down?

Reply:
As in, should we reference the allowed origins from the config?

Reply:
Sorry, that wasn't really clear now that I re-read what I wrote.

Well, is there any problem with locking websockets to just allow the FE?  Or is that going to be problematic?

e.g. `socketio.init_app(app, cors_allowed_origins=['https://dev.business.bcregistry.gov.bc.ca']`

If this works, I was thinking we make this a config.

Reply:
I'm not sure if that will work because the FE code runs in the user's browser. That URL is the web server that provides the code to the FE.

Reply:
I think it will be fine.  I haven't used websockets before though.

Based off my understanding, the websocket handshake will look like something below which will provide the origin that the web app is hosted on.  So I'm expecting example.com will be `dev.business.bcregistry.gov.bc.ca`

```
GET /chat HTTP/1.1
Host: example.com:8000
Upgrade: websocket
Connection: Upgrade
Origin: http://example.com
Sec-WebSocket-Key: dGhlIHNhbXBsZSBub25jZQ==
Sec-WebSocket-Version: 13
```

Reply:
Oh yeah? I just learned something new :)

Reply:
was testing a site that used websocket.  this is what i saw for the handshake to establish the websocket connection.

![image](https://github.com/bcgov/lear/assets/6685223/b453ca57-04f5-4d22-b2c7-aba988f6e0a8)

Reply:
We can most certainly add it as a config option

Reply:
ok, then maybe we can make the `cors_allowed_origins` value a configurable field that will be populated into a list.

e.g. `socketio.init_app(app, cors_allowed_origins=[<allowed_origins_placeholder>])`

_Examples of how config might look:_ 
WS_ALLOWED_ORIGINS = * 
OR
WS_ALLOWED_ORIGINS = dev.business.bcregistry.gov.bc.ca, https://dev.edit.business.bcregistry.gov.bc.ca, https://dev.create.business.bcregistry.gov.bc.ca